botcore/utils/commands.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from discord import Message
+from discord.ext.commands import Context, clean_content
+
+
+async def clean_text_or_reply(ctx: Context, text: Optional[str] = None) -> Optional[str]:
+    """Returns cleaned version of `text`, if given, else referenced message, if found, else `None`."""
```

Comment:
Please include a description for the paramaters and return in the docstring. If you aren't familiar, refer to the readme section on [writing docstrings](https://github.com/python-discord/bot-core/blob/main/docs/README.md#docstrings)

Reply:
:+1: